.eslintrc

```diff
@@ -60,7 +60,7 @@
     "guard-for-in": ["off"],
     "import/named": ["off"],
     "import/no-duplicates": ["error"],
-    "import/no-named-as-default": ["error"],
+    "import/no-named-as-default": ["off"],
```

Comment:
why do we want to turn this off?

Reply:
It looks like this has never been enforced, but now it is, probably because of webpack 4 upgrade. Here's the rule:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-named-as-default.md
We have this all over the place, our every export is named, and then we import it by that name. The rule seems reasonable, but it's too much work to follow it now.